content/blog/2018_workshop_attendees_useful_info.md

```diff
@@ -0,0 +1,39 @@
+Title: Useful info PyCon India Workshop Attendees
+Date: 2018-10-04 22:00
+Category: 2018
+Slug: useful-info-workshop-attendees
+
+
+If you are coming to PyCon India 2018 and attending workshops, then please read this blog to understand some dos and don'ts along with lot of useful info.
+<!-- PELICAN_END_SUMMARY -->
+
+Hope you're all set for PyCon India 2018.
+
+Here are a few things we would like to bring to your attention.
+
+## Reaching to Hyderabad:
+Please find information at https://in.pycon.org/2018/travel.html.
+
+## Reaching the venue (HICC):
+For attendees using public transport, shuttles will be available between Hitex Charminar (Arch)  and the venue - [HICC](https://goo.gl/maps/SrZKN3PhAau). For any further guidance, please reach out to our transport team @ Subhash - 9963754460
+
+## At the Registration Desk:
+The first step is to register and collect your kit from the front desk. Please carry your ticket as the team would be looking for the registration number on it.  The queues will be organized by Registration Number.
+![ticket image](https://github.com/pythonindia/inpycon2018/blob/master/img/ticket-image.jpeg)
+
+## During the conference:
```

Comment:
Can we make the heading here `During the workshops` ?

Reply:
Done.